bedrock/externalpages/templates/externalpages/pocket/includes/nav.html

```diff
@@ -1,13 +1,21 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
 {# This Source Code Form is subject to the terms of the Mozilla Public
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
+{% include 'externalpages/pocket/includes/mobile-nav.html' %}
+
 <header class="pocket-header">
     <nav class="global-nav-container">
-        <button class="global-nav-mobile-menu-btn inline">
+        <button class="global-nav-mobile-menu-btn inline" aria-labeledby="mobile-menu-icon-title" aria-describedby="mobile-menu-icon-description">
             <svg
                 class="hamburger-btn"
-                aria-labelledby="mobile-menu-menu-icon-title mobile-menu-menu-icon-description"
+                aria-labelledby="mobile-menu-icon-title mobile-menu-icon-description"
```

Comment:
When I test this with voiceover it reads "Open Open the Pocket Mobile menu, Button", so I think you can omit including `mobile-menu-icon-title` here?
```suggestion
                aria-labelledby="mobile-menu-icon-description"
```